cobalt/site/docs/development/setup-android.md

```diff
@@ -33,6 +33,8 @@ return and complete the following steps.
 
     If prompted, read and accept the license agreement.
 
+1.  Install [JDK 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html) and make sure it's before any other JDK on your PATH.
```

Comment:
Our linter doesn't cover markdown files but should still follow the 80 character line length limit. Can you split this line so no line is longer (links and commands are excepted if they are on their own lines)?

Reply:
done